packages/common/lib/types/CredentialIssuance.types.ts

```diff
@@ -1,11 +1,11 @@
 import { CredentialFormat, W3CVerifiableCredential } from '@sphereon/ssi-types';
-import {OpenId4VCIVersion} from "./OpenID4VCIVersions.types";
+
+import { OpenId4VCIVersion } from './OpenID4VCIVersions.types';
 
 export interface CredentialRequest {
   //TODO: handling list is out of scope for now
   type: string | string[];
-  //TODO: handling list is out of scope for now
-  format: CredentialFormat | CredentialFormat[];
+  format: CredentialFormat;
```

Comment:
Have you doublechecked this? I rather have us conform whenever we send out things, but be lenient when we get things

Reply:
it should be singular, but I reverted this for conformity